tests/projects/local-kit/kits/extra-1-kit/Cargo.toml

```diff
@@ -4,14 +4,8 @@ version = "0.1.0"
 edition = "2021"
 publish = false
 build = "../build.rs"
-# Don't rebuild crate just because of changes to README.
-exclude = ["README.md"]
 
 [package.metadata.build-kit]
-included-packages = [
-    "pkg-b",
-    "pkg-d",
-]
```

Comment:
An `included-packages` would be sort of gnarly to implement since it would need to inform the graph algorithm. When we talked about it we were thinking we would just go with Cargo dependencies defining the package list. `excluded-packages` could be a feature add fairly easily if anyone ends up needing that.